x/deployment/keeper/keeper_test.go

```diff
@@ -210,6 +223,41 @@ func Test_OnDeploymentClosed(t *testing.T) {
 	})
 }
 
+func Test_CloseGroup(t *testing.T) {
+	ctx, keeper := setupKeeper(t)
+	groups := createActiveDeployment(t, ctx, keeper)
+	gIdx0, gIdx1 := 0, 1
```

Comment:
why?

Reply:
Removed. I was thinking in terms of further referencing, but agree `group := group[0]` is a better way to access for these cases.